src/v/pandaproxy/context.h

```diff
@@ -11,8 +11,8 @@
 
 #pragma once
 
+#include "config/config_store.h"
```

Comment:
can you add a word or two about this config_store in the commit body. how is the decoupling related to the config_store?